### PR TITLE
Bug 2025955: Removing the extra indentation

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
From admingate_configmap.yaml

Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: install/0000_00_cluster-version-operator_01_admingate_configmap.yaml [hunk 1]
```diff
@@ -2,8 +2,7 @@ apiVersion: v1
 kind: ConfigMap
 data:
   ack-4.8-kube-1.22-api-removals-in-4.9: |
```
Comment (wking) [inline install/0000_00_cluster-version-operator_01_admingate_configmap.yaml:4]: From [the YAML spec][1]:

> Scalar content can be written in block notation, using a literal style (indicated by “|”) where all line breaks are significant. Alternatively, they can be written with the folded style (denoted by “>”) where each line break is folded to a space unless it ends an empty or a more-indented line.

So you could leave the following two lines alone, and move to `...: >` here, to get that internal line break folded into a space. 

[1]: https://yaml.org/spec/1.2.2/#23-scalars

Comment (LalatenduMohanty) [reply install/0000_00_cluster-version-operator_01_admingate_configmap.yaml]: I like the suggestion. Fixed it

**Review by wking — APPROVED** [review]
/lgtm

**Pull request merged**